everlive-push-hybrid/scripts/common.js

```diff
@@ -1,35 +1,60 @@
-(function (global) {
+(function(global) {
     'use strict';
 
     var app = global.app = global.app || {};
-    
+
     app.showAlert = function(message, title, callback) {
-        navigator.notification.alert(message, callback || function () {
-        }, title, 'OK');
+        navigator.notification.alert(message, callback || function() {}, title, 'OK');
     };
 
     app.showError = function(message) {
         app.showAlert(message, 'Error occured');
     };
-    
-    app.showLoading = function () {
+
+    app.showLoading = function() {
         app.mobile.showLoading();
     };
-    
-    app.hideLoading = function () {
+
+    app.hideLoading = function() {
         app.mobile.hideLoading();
     };
-    
-    app.isNullOrEmpty = function (value) {
+
+    app.isNullOrEmpty = function(value) {
         return typeof value === 'undefined' || value === null || value === '';
     };
 
-    app.isKeySet = function (key) {
+    app.isKeySet = function(key) {
         var regEx = /^\$[A-Z_]+\$$/;
         return !app.isNullOrEmpty(key) && !regEx.test(key);
     };
-    
-    app.getYear = function () {
+
+    app.isInSimulatorCheck = function() {
+        return window.navigator.simulator === true;
+    };
+
+    app.isInCompanionAppCheck = function() {
+        if (typeof(isInCompanionApp) !== 'undefined') {
```

Comment:
The proposed check does not work so will stay for now:
if (typeof(isInCompanionApp) !== 'undefined') {